### DISCUSSION TITLE
(ab)using the translation interface for a multi-site approach

### DISCUSSION THREAD

**Discussion by just-maik** [directus/directus#16103] [Feature Requests]
## How I currently do things
I configure a translation interface the following way:
![image](https://user-images.githubusercontent.com/16877165/197162497-dfcabace-fefc-42c5-a9b2-e915f9d1cd9a.png)
Using this Interface I can make fields translatable for website xy.
![image](https://user-images.githubusercontent.com/16877165/197162889-f154fcbe-05bb-4452-8426-1082b7b473a8.png)

## My problem with the current approach
When using a nested collection, directus doesn't memorize the previously selected key for the translation interface. This ruins the UX since the user has to re-select the translation key on every nested object opened.
![image](https://user-images.githubusercontent.com/16877165/197167867-f681582e-fa35-4afa-9dc0-21a967ff96b1.png)
(i disabled the backdrop-shadow so I can show my Issue)
## What I'd like to have
When using the translation interface I can set a check-mark on "use Current User Language".
Now since I don't use the languages table for the translation interface, this feature just doesn't work for me.
Is it possible to somehow set the field of the user that this default setting should relate to, so I can just create a "sites" field on "directus_users" and make it work.
A different approach may be simply drilling the selected translation key on nested collections into the drawer component, if the translation field, relates to the same language collection, at least in my monkey-brain.

For context, I'm currently using Directus to build a multi-tenant e-commerce/page-builder micro-service project.

I'd love to get some input on how to tackle this problem from the directus community 😇

**Comment by br41nslug** [comment]
Interesting approach 🤔 I have a question tho, do you need "site" translation interface to be nested? Having it only at the top level should be enough to create a separate content tree per site without having to select the linked site multiple times in the UI.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
